packages/plugins/content/image/package.json

```diff
@@ -17,12 +17,12 @@
     "flow:generate": "flow-typed install  --flowVersion=0.46.0"
   },
   "peerDependencies": {
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2",
-    "react-tap-event-plugin": "^2.0.1"
+    "react": "^16.0.0",
+    "react-dom": "^16.0.0"
   },
   "dependencies": {
-    "material-ui": "^0.19.1",
+    "@material-ui/core": "^1.2.1",
```

Comment:
Is core always required?

Reply:
Yes material-ui/core is now the main package

Reply:
Ok, I'm asking because I don't see any files requiring from `@material-ui/core`

Reply:
@material-ui/icons appears to depend on @material-ui/core and failed to compile without it.

Reply:
Thank you for checking! Then this is 👌